packages/wallets/ambire/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import { WalletAdapterType } from '@reef-knot/types';
+import { getWalletConnectConnector } from '@reef-knot/wallets-helpers';
+import WalletIcon from './icons/ambire.svg';
+
+export const Ambire: WalletAdapterType = ({ rpc }) => {
```

Comment:
I would personally prefer here to do like `() => ({ walletName:... })` but you are the author, so just sharing opinion